### DISCUSSION TITLE
selinux compilation error

### DISCUSSION THREAD

**Discussion by importMan** [SELinuxProject/selinux#453] [Q&A] [answered]
![1732850611085](https://github.com/user-attachments/assets/c87610c1-1e36-4f97-80cb-22651e602eb4)

![1732850683213](https://github.com/user-attachments/assets/14268d54-95f0-43a1-ac1b-83e83917d293)

I keep getting errors similar to those in the above picture. Is it because I am using the wrong library version?

**Comment by cgzones** [comment, marked as answer]
You need to build against the in-tree libsepol version, e.g. by setting the `DESTDIR` environment variable and running `make install`, which will install libsepol headers into *DESTDIR* and append the necessary CFLAGS and LDFLAGS paths.
